media-libs/raylib/raylib-1.9.7.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-multilib
+
+DESCRIPTION="A simple and easy-to-use library to learn videogames programming"
+HOMEPAGE="http://www.raylib.com/"
+
+LICENSE="ZLIB"
+IUSE="alsa examples games static-libs system-glfw wayland"
+SLOT="0"
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/raysan5/raylib.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/raysan5/raylib/archive/${PV}-dev.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/raylib-${PV}-dev"
+fi
+
+RDEPEND="
+	alsa? ( media-libs/alsa-lib )
+	!wayland? (
```

Comment:
This does not match the common use of `USE=wayland` (which is supposed to give Wayland support without blocking X11), and so people who have `USE=wayland` but use X11 will be surprised by this package not working. If you really need to make this exclusive between X11 and Wayland, please have explicit flags like `IUSE="wayland X"`, and `REQUIRED_USE="^^ ( X wayland )"` to force user to explicitly choose.